test/notification.js

```diff
@@ -206,18 +205,18 @@ describe('<Notification />', () => {
     }, mockNotification.dismissAfter);
   });
 
-  it('onDismiss fires correctly without prop change', done => {
```

Comment:
I think you just changed `'onDismiss fires correctly without prop change'` to `'onDismiss fires once when dismissAfter is passed'`. That's fine.